spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/direct/BigQueryRDDContext.java

```diff
@@ -117,7 +117,8 @@ public scala.collection.Iterator<InternalRow> compute(Partition split, TaskConte
               readSession.getAvroSchema().getSchema(),
               Optional.of(schema),
               Optional.of(tracer),
-              SchemaConvertersConfiguration.from(options));
+              SchemaConvertersConfiguration.from(options),
+              options.getResponseCompressionCodec());
     } else {
       converter =
           ReadRowsResponseToInternalRowIteratorConverter.arrow(
```

Comment:
Shouldn't this be enabled to the arrow type as well?

Reply:
I am decompressing in ReadRowsResponseInputStreamEnumeration.  Do I also need to do it here, or would that be attempting decompression twice?